wire-ios-data-model/Use cases/LocalConversationRemovalUseCase.swift

```diff
@@ -0,0 +1,59 @@
+////
+// Wire
+// Copyright (C) 2023 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+import WireSystem
+
+public protocol LocalConversationRemovalUseCaseProtocol {
+    func removeConversation(_ conversation: ZMConversation, syncContext: NSManagedObjectContext)
+}
+
+public class LocalConversationRemovalUseCase: LocalConversationRemovalUseCaseProtocol {
+
+    public init() {}
+
+    public func removeConversation(
```

Comment:
As the use case does only one action of its name we can just name it with less context
```suggestion
    public func invoke(
```

Reply:
done